windows/helpers/phase3/install_ci_uploader.ps1

```diff
@@ -0,0 +1,22 @@
+param (
+    [Parameter(Mandatory=$true)][string]$Version,
+    [Parameter(Mandatory=$true)][string]$Sha256
+)
+
+$ErrorActionPreference = 'Stop'
+
+$isInstalled, $isCurrent = Get-InstallUpgradeStatus -Component "datadog-ci" -Keyname "version" -TargetValue $Version
+if($isInstalled -and $isCurrent) {
+    Write-Host -ForegroundColor Green "Datadog-ci uploader up to date"
+    return
+}
+if($isInstalled -and -not $isCurrent){
+    Remove-Item -Recurse -Force c:\datadog-ci -ErrorAction SilentlyContinue
+}
+$source="https://github.com/DataDog/datadog-ci/releases/download/v${Version}/datadog-ci_win-x64"
+$target = "c:\datadog-ci"
```

Comment:
I'm not used to Windows but isn't it in uppercase usually ?
```suggestion
$target = "C:\datadog-ci"
```

Reply:
usually by convention yes, but paths are technically case insensitive